### PR TITLE
adding (C++)

### PR DESCRIPTION
**Pull request by itsOmSarraf**

### CODE REVIEW

**Comment by ericspring08** [comment]
Could you please add C++ to README.md along with its description alphabetically.

File: Languages/Cplusplus.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- **Links** - [Official Website](https://isocpp.org/)
+- **Applications**
+  * Operating Systems
+  * GUI Based Applications
+  * Web Browsers
+  * Games
+  
+  
+## Package Managment
+* There's no package manager for C++ libraries
```
Comment (ericspring08) [inline Languages/Cplusplus.md:14]:
```suggestion
* [conan](https://conan.io/)
```

Comment (itsOmSarraf) [reply Languages/Cplusplus.md]: okay thanks

**Comment by ericspring08** [comment]
Could you please add C++ to README.md along with its description alphabetically.

**Pull request merged**